crates/cli/src/subcommands/identity.rs

```diff
@@ -309,12 +305,6 @@ async fn exec_remove(mut config: Config, args: &ArgMatches) -> Result<(), anyhow
         return Err(anyhow::anyhow!("Must provide an identity or name to remove"));
     }
 
-    if force && !(all || all_server.is_some()) {
-        return Err(anyhow::anyhow!(
-            "The --force flag can only be used with --all or --all-server"
-        ));
-    }
```

Comment:
I removed this check since the `--yes` param is a bit more generic than `--force` was. It's kind of fine to provide it when it won't make a difference, since it's just a general "skip any confirmations if they come up" flag now.